configuration/persistence.md

```diff
@@ -11,59 +11,73 @@ openHAB can store data over time; this is known as persistence.
-Scroll down to "Persistence", and select your Default Service from the drop-down list.
+Scroll down to "Persistence", and select your default service from the drop-down list.
 Note that you must first install a persistence add-on before you make this selection.
 Be sure to save your choice once you have selected your default service.
 
-## Configuration
+## Persistence Configuration
 
-Persistence Strategies are configured in a file named `<persistenceservice>.persist`, stored in `$OPENHAB_CONF/persistence` ("persistenceservice" is replaced by the name of your add-on (e.g. `rrd4j.persist`)).
+Persistence Strategies are configured in a file named `<persistenceservice>.persist`, stored in `$OPENHAB_CONF/persistence` (Replace "persistenceservice" with the name of your persistence add-on (e.g. `rrd4j.persist`)).
 
-Strategies may be used to persist an Item state when some event has occurred (e.g. an Item state has been updated or changed), on a time schedule, or at a specific time of day (e.g. through a [cron expression](http://www.quartz-scheduler.org/documentation/quartz-2.1.x/tutorials/crontrigger).
+### Persistence Triggers
 
-Persistence configuration files consist of several sections:
+The persistence of an Item's state may be triggered when that Item changes state or when the Item is updated (even if its state did not change).
+Persistence may also be triggered by a time-related event (see Cron Persistence Triggers below).
 
-### Strategies section
+### Strategies
 
-This section allows you to define strategies and to declare a set of default strategies to use for this persistence service.  The syntax is as follows:
+This section allows you to define one or more `Strategies` that will be applied to `Items` (you specifiy the `Items` in the `Items` section).
+This section also defines a default strategy.
+The syntax is as follows:
 
 ```java
 Strategies {
   <strategyName1> : "cronexpression1>"
+  <strategyName2> : "cronexpression2"
+  ...
+  default = everyChange
   ...
 }
 ```
 
-The following strategies are defined internally:
+The `default` parameter assigns a strategy to be used if one is not specified in the `Items` section below.
+The `default` parameter may be omitted from the `Strategies` section, but only if a strategy is provided in each line of the `Items` section.
+
+#### Predefined Strategies
+
+The following strategies are defined internally and may be used in place of `strategyName` above:
 
 - everyChange: persist the Item state whenever its state has changed
 - everyUpdate: persist the Item state whenever its state has been updated, even if it did not change
-- restoreOnStartup: load and initialize the last persisted Item state on openHAB startup (if the Item state is undefined (`UNDEF`)).  See below.
+- restoreOnStartup: load and initialize the last persisted state of the Item on openHAB startup (if the Item state is undefined (`UNDEF`)).  See below.
```

Comment:
"`restoreOnStartup` will load and ..." ?

Reply:
Well, the way it is currently written (this is the original language from the Wiki) it matches the other points.  RE:

everyChange: persist the...
everyUpdate: persist the...
restoreOnStartup: load and initialize the...

I could change as follows:

restoreOnStatup: will load and initialize the...

but then I think the others should follow the same form:

everyChange: will persist the...
everyUpdate: will persist the...

My two cents are that the "will" is extra, but I am happy to put it in if you like.  But I do feel they should all be the same.

Reply:
Sorry, this is not the first time I'm misleading you with super short comments.

1. Yes all three
2. Please add code fences
3. I'm fine with either of
  * `restoreOnStartup`: load...
  * `restoreOnStartup` - load...
  * `restoreOnStartup` will load...

Up to you!

Reply:
OK.  Not misleading.  But...

Two of them cause a persistence action.  The last one causes a restore action.  So 1 & 2 should be "persist" or "store".  Number 3 should be "load".  Agree?